### DISCUSSION TITLE
Accessibility of legally protected data

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#32] [General]
There are very interesting population data, telephone data, which are not public data and can be accessed by purchasing them at a high price.

These data are of great importance in case of emergencies. Therefore, policy measures should be put in place for public access to these data, after anomization of the data.

**Comment by thorsten-reitz** [comment]
I would like to point out that establishing the European data spaces, as outlined in the data strategy, provides a possible soluton for this issue.

**Comment by naomithiru** [comment]
For the Tracing use-case, local sewer datasets tend to not be open/publicly available. A solution for this would be limited data exposed in a tracing result. If restricted datasets are made available for tracing, the tracing action can be performed, with the trace results restricting information to the minimum allowed/agreed upon features to maintain privacy.

**Comment by AndersOstmanNovogit** [comment]
This issue is reported in section 4.2.4 in D4.3

The Data Governance Act is the European legislation aiming to foster the availability of data for use by increasing trust in data intermediaries and by strengthening data-sharing mechanisms across the EU. The instrument intend to make protected public sector data available for re-use, including data that might be subject to data protection legislation, intellectual property, or contain trade secrets or other commercially sensitive information. This legislation is still at the proposal stage but it might be one solution for future problems of this type. The creation of European data spaces is one type of development supporting the implementation of the Data Governance Act.

Another possible solution is to sign agreements with the data provider, assuring that the source data will not be shared with anyone, only the final result. If required, final approval of the data provider might be needed for sharing the final result.

Data may also be resampled to a lower resolution, thereby avoiding inappropriate usage. For instance, in cases where personal integrity is the reason for access restrictions, aggregated personal data may be used if no specific individual can be spotted in the data sets. In other cases, synthetic population may be used, having the same statistical properties as the real data set.

**Reply by naomithiru** [reply]
> Another possible solution is to sign agreements with the data provider, assuring that the source data will not be shared with anyone, only the final result. If required, final approval of the data provider might be needed for sharing the final result.

This is the recommendation we put forth as well, where the datasets are made available to prepare a tracing network, but sensitive information is restricted in the tracing result in accordance with agreements the data providers.

**Reply by AndersOstmanNovogit** [reply]
Now moved to D4.2 section 4.1.2

**Comment by sMorrone** [comment]
Regarding the Data Governance Act, this could ONLY to a certain extent be seen as a possible solution to the problem. I mean, the act goes  not so much in the direction of unrestricting datasets, but in making clear who and under what conditions (e.g., signing specific EU-common sectorial agreements) can have access to data + making available a secure environment for data exchange.
